### DISCUSSION TITLE
AUTH_CODE Explanation

### DISCUSSION THREAD

**Discussion by j-funk** [nicumicle/simple-jwt-login#5] [Q&A] [answered]
Hello, I wanted to find out more about the AUTH_CODE parameter and how to use it.  It's not fully explained how to use it / set up.  Except for some warning about not registering new users without requiring it is 'not recommended'.  Would you mind explaining the benefits of the AUTH_CODE?

Thanks for this great resource.

**Comment by lexicon2008** [comment, marked as answer]
I am sure the AUTH CODE is meant to keep your request private... so generally people do not just pick your public codes generate tokens to make  requests, so the AUTH CODE acts like a 2nd verification to the request

**Comment by nicumicle** [comment]
Hello, 

Here is small documentation about AUTH CODES: 

https://docs.simplejwtlogin.com/#/?id=auth-codes

Let me know if you have other questions. :)
